src/main/java/salesforce/utils/SheetManager.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package salesforce.utils;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+
```

Comment:
imports are not necessary separate

please press ctrl + alt + O to the identation of the comments